chaoscenter/authentication/api/handlers/rest/project_handler.go

```diff
@@ -150,6 +149,35 @@ func getInvitation(service services.ApplicationService, member entities.MemberIn
 	return "", nil
 }
 
+// ListInvitations returns the Invitation status
+func ListInvitations(service services.ApplicationService) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		uID := c.MustGet("uid").(string)
+		var response []entities.ListInvitationResponse
+		projects, err := service.ListInvitations(uID)
+		if err != nil {
+			log.Error(err)
```

Comment:
Can you add a more descriptive log here 
```suggestion
			log.Errorf("Error while getting invitations: %v", err)
```